### ISSUE TITLE
Adjustments to the size and order of the columns in the report view isn't saved between sessions

### ISSUE THREAD

**Issue by IndexLibrorumProhibitorum** [magge-faf/faf-moderator-client-develop-modified#34]
![image](https://github.com/magge-faf/faf-moderator-client-develop-modified/assets/64420140/a531f018-8f23-402a-967c-82d09b16f6f3)
The columns in the report view, as shown above, can be reordered and resized. This is very useful for me, because I am working with a limited size screen and all the important information isn't on the screen without some shuffling and resizing.

However, these changes are not saved between sessions, and seem to reset randomly during the session as well. (Haven't been able to replicate this, so not 100% sure if that is correct.) 

I would like these changes to be remembered between sessions.

**Comment by magge-faf** [comment]
What is your screen size? 1080p?

**Comment by IndexLibrorumProhibitorum** [comment]
2736 x 1824

**Comment by magge-faf** [comment]
@IndexLibrorumProhibitorum I tried for two sessions ~40 minutes to make it happen via Preferences and failed to save/load the FXML values for the size/order of columns. 

Copilot AI was suggesting some solutions, but I could not estimate the ETA via its approach, and had no idea where it was heading to at the end. It could have worked perfectly "in the next minutes" or "never" via its support.

Well, however - I know how to save/load the prefWidth values via JSON, (Aka: If I only know how to use a hammer, then everything looks like a nail, *cough*)

I went that road to have the feature partially working for customizable sizes via JSON. The column-order is more tricky, and takes too much time to make it happen right now.

Basically, you enter the values in the JSON TableColumnsWidthSettings.json located in the folder ConfigurationModerationToolFAF:

That handles the columns sizes for columns in Reports Tab. If the order-thing is crucial, I can take another look in the future.

Not an optimal solution, but it will do it for now.

![image](https://github.com/magge-faf/faf-moderator-client-develop-modified/assets/101107758/6c01b366-ef37-43f0-a185-c0c6b1ff7c3b)

![image](https://github.com/magge-faf/faf-moderator-client-develop-modified/assets/101107758/076b4c94-9434-43a5-8b7a-b60e47b97587)
